operator/pkg/apis/operator/v1alpha1/helper.go

```diff
@@ -0,0 +1,18 @@
+package v1alpha1
+
+import (
+	"fmt"
+)
+
+// CreateImage return image object. the component is represents karmada components name.
+func CreateImage(repository, component, tag string) Image {
+	return Image{
+		ImageRepository: fmt.Sprintf("%s/%s", repository, component),
+		ImageTag:        tag,
+	}
+}
+
+// String retrun the image name.
+func (image *Image) String() string {
```

Comment:
I guess here you want to implement the `Stringer` interface, but we can't use a `pointer` receiver because it only takes effect with a pointer of that type.

I suggest renaming this to `Name()` and using the type as the receiver.

Reply:
It makes sence to me.